### PR TITLE
bpo-39509: Update HTTP status code to follow IANA

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue39509

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2020-02-01-00-03-06.bpo-39509.-YxUSf.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+HTTP status code ``103 EARLY_HINTS`` and ``425 TOO_EARLY`` are added to
+:class:`http.HTTPStatus` Patch by Dong-hee Na.
```
Comment (BoboTiG) [inline Misc/NEWS.d/next/Library/2020-02-01-00-03-06.bpo-39509.-YxUSf.rst]: Small improvement (missing 's' and dot):
> HTTP status codes ``103 EARLY_HINTS`` and ``425 TOO_EARLY`` are added
to :class:`http.HTTPStatus`. Patch by Dong-hee Na.

File: Doc/whatsnew/3.9.rst [hunk 2]
```diff
@@ -186,6 +186,12 @@ Added a new function :func:`gc.is_finalized` to check if an object has been
 finalized by the garbage collector. (Contributed by Pablo Galindo in
 :issue:`39322`.)
 
+http
+----
+
+HTTP status code ``103 EARLY_HINTS`` and ``425 TOO_EARLY`` are added to
```
Comment (BoboTiG) [inline Doc/whatsnew/3.9.rst]:
```suggestion
HTTP status codes ``103 EARLY_HINTS`` and ``425 TOO_EARLY`` are added to
```

**Review by BoboTiG — CHANGES_REQUESTED** [review]
Thank you for the patch @corona10 :+1:

**Comment by corona10** [comment]
@BoboTiG  Thanks for the review, I 've update the PR :)

**Comment by corona10** [comment]
@rhettinger @vadmium Can you please take a look?

**Review by BoboTiG — APPROVED** [review]

**Comment by vadmium** [comment]
Looks okay to me

**Comment by corona10** [comment]
@vadmium 

If you don't mind, can we merge this PR?, I don't have merge permission :)

**Comment by vadmium** [comment]
Sorry, I don't really have the time or motivation to merge anything these days. Hopefully you can find someone else.

**Comment by corona10** [comment]
@vstinner Can you please take a look at this?

**Comment by vstinner** [comment]
cc @JulienPalard @matrixise

**Review by serhiy-storchaka — APPROVED** [review]

**Pull request merged**